packages/ember-cli-fastboot/.eslintrc.js

```diff
@@ -15,6 +15,8 @@ module.exports = {
     browser: true
   },
   rules: {
+    'ember/no-get': 'warn',
```

Comment:
Adding a new ember app to the test-packages has updated `ember-cli-eslint` which has added new linting rules.